  Exclude some packages to speed up the build process and reduce install size.

--- a/packages/fpmake_add.inc	2020-09-09 17:22:17.000000000 +0200
+++ b/packages/fpmake_add.inc	2022-12-15 13:01:44.982641453 +0100
@@ -35,9 +35,8 @@
   add_fftw(ADirectory+IncludeTrailingPathDelimiter('fftw'));
   add_fpgtk(ADirectory+IncludeTrailingPathDelimiter('fpgtk'));
   add_fpindexer(ADirectory+IncludeTrailingPathDelimiter('fpindexer'));
   add_fpmkunit(ADirectory+IncludeTrailingPathDelimiter('fpmkunit'));
-  add_fppkg(ADirectory+IncludeTrailingPathDelimiter('fppkg'));
   add_fv(ADirectory+IncludeTrailingPathDelimiter('fv'));
   add_gdbint(ADirectory+IncludeTrailingPathDelimiter('gdbint'));
   add_gdbm(ADirectory+IncludeTrailingPathDelimiter('gdbm'));
   add_ggi(ADirectory+IncludeTrailingPathDelimiter('ggi'));
@@ -135,13 +134,11 @@
   add_zorba(ADirectory+IncludeTrailingPathDelimiter('zorba'));
   add_Google(ADirectory+IncludeTrailingPathDelimiter('googleapi'));
   add_fcl_pdf(ADirectory+IncludeTrailingPathDelimiter('fcl-pdf'));
   add_odata(ADirectory+IncludeTrailingPathDelimiter('odata'));
-  add_pastojs(ADirectory+IncludeTrailingPathDelimiter('pastojs'));
   add_libgc(ADirectory+IncludeTrailingPathDelimiter('libgc'));
   add_libfontconfig(ADirectory+IncludeTrailingPathDelimiter('libfontconfig'));
   add_fcl_report(ADirectory+IncludeTrailingPathDelimiter('fcl-report'));
   add_webidl(ADirectory+IncludeTrailingPathDelimiter('webidl'));
   add_gnutls(ADirectory+IncludeTrailingPathDelimiter('gnutls'));
-  add_ide(ADirectory+IncludeTrailingPathDelimiter('ide'));
   add_vclcompat(ADirectory+IncludeTrailingPathDelimiter('vcl-compat'));
   

--- a/utils/fpmake_add.inc	2018-08-10 09:12:49.000000000 +0200
+++ b/utils/fpmake_add.inc	2022-12-15 13:20:38.891021857 +0100
@@ -1,20 +1,12 @@
   add_dxegen(ADirectory+IncludeTrailingPathDelimiter('dxegen'));
   add_fpcm(ADirectory+IncludeTrailingPathDelimiter('fpcm'));
-  add_fpcmkcfg(ADirectory+IncludeTrailingPathDelimiter('fpcmkcfg'));
   add_fpcres(ADirectory+IncludeTrailingPathDelimiter('fpcres'));
   add_fpcreslipo(ADirectory+IncludeTrailingPathDelimiter('fpcreslipo'));
-  add_fpdoc(ADirectory+IncludeTrailingPathDelimiter('fpdoc'));
   add_fpmc(ADirectory+IncludeTrailingPathDelimiter('fpmc'));
-  add_fppkg_util(ADirectory+IncludeTrailingPathDelimiter('fppkg'));
   add_fprcp(ADirectory+IncludeTrailingPathDelimiter('fprcp'));
-  add_h2pas(ADirectory+IncludeTrailingPathDelimiter('h2pas'));
   add_json2pas(ADirectory+IncludeTrailingPathDelimiter('json2pas'));
   add_importtl(ADirectory+IncludeTrailingPathDelimiter('importtl'));
-  add_instantfpc(ADirectory+IncludeTrailingPathDelimiter('instantfpc'));
-  add_pas2fpm(ADirectory+IncludeTrailingPathDelimiter('pas2fpm'));
   add_pas2jni(ADirectory+IncludeTrailingPathDelimiter('pas2jni'));
-  add_pas2ut(ADirectory+IncludeTrailingPathDelimiter('pas2ut'));
   add_rmwait(ADirectory+IncludeTrailingPathDelimiter('rmwait'));
   add_tply(ADirectory+IncludeTrailingPathDelimiter('tply'));
   add_unicode(ADirectory+IncludeTrailingPathDelimiter('unicode'));
-  add_pas2js(ADirectory+IncludeTrailingPathDelimiter('pas2js'));
